requirements.txt

```diff
@@ -1,4 +1,4 @@
-nypl-py-utils[avro-client]==1.2.0
+avro
+nypl-py-utils[avro-client,config-helper]==1.2.0
 pybase64
-python-csv
-python-io
\ No newline at end of file
+requests
```

Comment:
`requests` is included in `avro-client`

Reply:
Added requests back in since lambda_function is failing without in (while testing in QA)

Reply:
I'm worried this means something's not working right -- what's the error you get?

Reply:
Something along the lines of `Module lambda_function unable to find requests`